JuiceMaker/JuiceMaker/Controller/OrderJuiceViewController.swift

```diff
@@ -14,71 +14,83 @@ final class OrderJuiceViewController: UIViewController {
     @IBOutlet weak private var kiwiStockLabel: UILabel!
     @IBOutlet weak private var mangoStockLabel: UILabel!
     
+    @IBOutlet weak private var ddalbaJuiceOrderButton: UIButton!
+    @IBOutlet weak private var strawberryJuiceOrderButton: UIButton!
+    @IBOutlet weak private var bananaJuiceOrderButton: UIButton!
+    @IBOutlet weak private var pineappleJuiceOrderButton: UIButton!
+    @IBOutlet weak private var kiwiJuiceOrderButton: UIButton!
+    @IBOutlet weak private var mangoJuiceOrderButton: UIButton!
+    @IBOutlet weak private var mangkiJuiceOrderButton: UIButton!
+    
     private let juiceMaker = JuiceMaker()
-   
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
         updateStockLabel()
+        setUpNotificationObserver()
     }
     
     private func updateStockLabel() {
-        strawberryStockLabel.text = juiceMaker.currentFruitStock(of: .strawberry)
-        bananaStockLabel.text = juiceMaker.currentFruitStock(of: .banana)
-        pineappleStockLabel.text = juiceMaker.currentFruitStock(of: .pineapple)
-        kiwiStockLabel.text = juiceMaker.currentFruitStock(of: .kiwi)
-        mangoStockLabel.text = juiceMaker.currentFruitStock(of: .mango)
+        strawberryStockLabel.text = juiceMaker.currentFruitStock(of: .strawberry).description
+        bananaStockLabel.text = juiceMaker.currentFruitStock(of: .banana).description
+        pineappleStockLabel.text = juiceMaker.currentFruitStock(of: .pineapple).description
+        kiwiStockLabel.text = juiceMaker.currentFruitStock(of: .kiwi).description
+        mangoStockLabel.text = juiceMaker.currentFruitStock(of: .mango).description
     }
     
-    @IBAction private func orderJuiceButtonTapped(_ sender: UIButton) {
-        switch sender.tag {
-        case 1:
-            completeOrder(of: .strawberryBananaJuice)
-        case 2:
-            completeOrder(of: .mangoKiwiJuice)
-        case 3:
-            completeOrder(of: .strawberryJuice)
-        case 4:
-            completeOrder(of: .bananaJuice)
-        case 5:
-            completeOrder(of: .pineappleJuice)
-        case 6:
-            completeOrder(of: .kiwiJuice)
-        case 7:
-            completeOrder(of: .mangoJuice)
-        default:
-            print("선택오류")
-        }
+    private func setUpNotificationObserver() {
+        NotificationCenter.default.addObserver(self, selector: #selector(self.didDismissManageStockView(_:)), name: NSNotification.Name("dismiss"), object: nil)
     }
```

Comment:
```suggestion
    private func setUpNotificationObserver() {
        NotificationCenter.default.addObserver(self, 
        selector: #selector(self.didDismissManageStockView(_:)),
        name: NSNotification.Name("dismiss"), 
        object: nil)
    }
```

코드 한 줄이 길다면 이렇게 매개변수를 기준으로 개행을 넣어주기도 합니다 🙂
(위 코드에는 여백이 표현되진 않았지만 매개변수가 같은 열에 있도록 정렬될 겁니다!)